### ISSUE TITLE
Results from paper on Brats2018 (Table 1)

### ISSUE THREAD

**Issue by samra-irshad** [weningerleon/BraTS2018#3]
Hi Leon, thanks for sharing your research with us.

I am confused about one thing in paper, the results mentioned for segmentation performance in Table 1 are based on actual validation and test set provided by challenge organizers or are they based on training set (partitioned into test, validation and training subsets)?

**Comment by weningerleon** [comment]
Thank you for your question, I apologize if the Table caption is slightly confusing:
Line 1 in the table is for the training set (cross-validated), while line 2 is for the validation set, provided by the challenge organizers.
If I remember correctly, at time of writing, the test set results were not provided.
